.github/workflows/build-images-action.yml

```diff
@@ -44,3 +44,13 @@ jobs:
             "BUILD_CONTAINER_IMAGE_GIT_REFERENCE": "${{ github.ref }}"
           }
         job_timeout: "1000"
+    - name: Slack Notification on Failure
+      if: ${{ failure() }}
+      uses: rtCamp/action-slack-notify@v2.0.2
+      env:
+        SLACK_TITLE: 'GitHub Action Failed in ${{ github.repository }}'
```

Comment:
Would be useful to get the full url of the build too. I imagine something like this would work, but I have not tested it out:
```
${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Reply:
Nvm. I found out that the action already has a field for full url.